plutus-tx-plugin/test/Budget/9.6/matchAsDataE.pir.golden

```diff
@@ -18,8 +18,8 @@
            Tuple2
              {integer}
              {b}
-             (fstPair {integer} {b} tup)
-             (sndPair {integer} {b} tup))
+             (case integer tup [(\(l : integer) (r : b) -> l)])
+             (case b tup [(\(l : integer) (r : b) -> r)]))
```

Comment:
There's of course the issue that we want to only `case` on `tup` once, i.e. we currently generate sub-optimal code.

Not for this PR though, should be addressed in a follow-up.